plugins/resources/compound/libcompound.cpp

```diff
@@ -138,6 +137,71 @@ namespace
 
         return nullptr;
     }
+
+    // returns a std::tuple with:
+    //  - cache_replica_number
+    //  - cache_replica_status
+    //  - archive_replica_number
+    //  - archive_replica_status
+    auto get_replica_status_and_number_for_cache_and_archive(irods::plugin_context& _ctx)
```

Comment:
naming... flip the order to match the tuples?

get_replica_status_and_number_for_cache_and_archive -> get_replica_number_and_status_for_cache_and_archive

Reply:
done